_includes/tutorials/hopping-windows/ksql/markup/dev/transient-window.adoc

```diff
@@ -0,0 +1,13 @@
+Now here comes the fun part. We need to create a query capable of detecting when the temperature drops below 45F for a period of 10 seconds. However, since our sensor emits readings every 5 seconds, we need to come up with a way to detect this even if the drop goes beyond the interval of 10 seconds. Hence why we are using hopping windows for this scenario. Since hopping windows are possibly-overlapping, we may have situations in which the temperature dropped across multiple 10 second windows.
```

Comment:
```suggestion
Now here comes the fun part. We need to create a query capable of detecting when the temperature drops below 45 °F for a period of 10 seconds. However, since our sensor emits readings every 5 seconds, we need to come up with a way to detect this even if the drop goes beyond the interval of 10 seconds. Hence why we are using hopping windows for this scenario. Since hopping windows may overlap, we may have situations in which the temperature dropped across multiple 10 second windows.
```